docs/hopsworksai/azure/aks_acr_integration.md

```diff
@@ -0,0 +1,229 @@
+Go to kubernetes services in the azure portal and click add. Place the kubernetes cluster in the same resource group as the hopsworks cluster will reside in and choose a name for the kubernetes cluster.
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/azure/aks-base.png">
+      <img src="../../../assets/images/hopsworksai/azure/aks-base.png" alt="AKS general configuration">
+    </a>
+    <figcaption>AKS general configuration</figcaption>
+  </figure>
+</p>
+
+Next click the Authentication tab and verify the settings are identical to the picture below.
+!!! note
+Currently AKS is only supported through managed identities. 
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/azure/aks-base.png">
+      <img src="../../../assets/images/hopsworksai/azure/aks-base.png" alt="AKS authentication configuration">
+    </a>
+    <figcaption>AKS authencation configuration</figcaption>
+  </figure>
+</p>
+
+Next go to the networking tab and check Azure CNI. The portal will automatically fill in the IP address ranges for the kubernetes virtual network. Take note of the virtual network name that is created, in this example the virtual network name was hopsworksstagevnet154. Lastly, check the private cluster option. 
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/azure/aks-base.png">
+      <img src="../../../assets/images/hopsworksai/azure/aks-base.png" alt="AKS network configuration">
+    </a>
+    <figcaption>AKS network configuration</figcaption>
+  </figure>
+</p>
+
+Next go to integrations. Under container registry click Create new. 
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/azure/aks-acr-create.png">
+      <img src="../../../assets/images/hopsworksai/azure/aks-acr-create.png" alt="AKS create ACR">
+    </a>
+    <figcaption>AKS create ACR</figcaption>
+  </figure>
+</p>
+
+Choose a name for the registry and check the SKU to be premium. Then press OK. 
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/azure/aks-acr.png">
+      <img src="../../../assets/images/hopsworksai/azure/aks-acr.png" alt="AKS ACR configuration">
+    </a>
+    <figcaption>ACR configuration</figcaption>
+  </figure>
+</p>
+
+Next press review and create, then create the cluster.
+
+Because the kubernetes API service is private the hopsworks cluster must be able to reach it over a private network. There are two options to integrate with a private AKS cluster. The first option is to put the hopsworks cluster in a pre-defined virtual network with peering setup to the kubernetes network. The second option is to create a subnet inside the kubernetes virtual network where the hopsworks cluster will be placed.
+
+#### Step 1 option a Peering setup
+
+In order to create a peering first a virtual network for the hopsworks cluster must be setup. Go to virtual networks and press add.
```

Comment:
In order to establish virtual peering between the kubernetes cluster and Hopsworks, you need to select or create a virtual network for Hopsworks.

Note: Hopsworks, not hopsworks. This mistake is in many places in the text.

Reply:
Muscle memory from coding. I have capitalized Hopsworks everywhere in the file.